src/ray/util/exponential_backoff.h

```diff
@@ -33,11 +51,28 @@ class ExponentialBackoff {
   /// @return the delay in ms based on the formula
   static uint64_t GetBackoffMs(uint64_t attempt,
                                uint64_t base_ms,
-                               uint64_t max_backoff_ms = kMaxBackoffMs);
+                               uint64_t max_backoff_ms = kDefaultMaxBackoffMs);
+
+  uint64_t Next() {
+    auto ret = curr_value_;
+    curr_value_ = curr_value_ * multiplier_;
+    curr_value_ = std::min(curr_value_, max_value_);
+    return ret;
+  }
+
+  uint64_t Current() { return curr_value_; }
+
+  void Reset() { curr_value_ = initial_value_; }
```

Comment:
Can we split unifying ExponentialBackoff to a separate PR?

Reply:
https://github.com/ray-project/ray/pull/49957